### PR TITLE
[SMALLFIX] stop sync after failing over

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java [hunk 1]
```diff
@@ -446,4 +446,19 @@ public void setTxId(long mountId, long txId) {
   public ExecutorService getExecutor() {
     return mExecutorService;
   }
+
+  /**
+   * Stop the sync manager.
+   */
+  public void stop() {
+    try {
+      for (long mountId : mFilterMap.keySet()) {
```
Comment (gpang) [inline core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java]: Should the for loop be outside the try, so we try to stop all the mounts?

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 2]
```diff
@@ -608,6 +608,7 @@ public void stop() throws IOException {
       mAsyncAuditLogWriter = null;
     }
     super.stop();
+    mSyncManager.stop();
```
Comment (gpang) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java]: does this have to be stopped before `super.stop()`?

File: core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java [hunk 3]
```diff
@@ -446,4 +446,19 @@ public void setTxId(long mountId, long txId) {
...
+
+  /**
+   * Stop the sync manager.
+   */
+  public void stop() {
+    for (long mountId : mFilterMap.keySet()) {
+      try {
+        stopSyncForMount(mountId);
+      } catch (IOException e) {
+        LOG.info("IOException encountered in stopping activeSyncManager {}", e);
```
Comment (gpang) [inline core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java]: for both of these, can we log the mount id and the mount URI?

File: core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java [hunk 4]
```diff
@@ -446,4 +446,19 @@ public void setTxId(long mountId, long txId) {
...
+      } catch (InvalidPathException e) {
+        LOG.info("InvalidPathException encountered in stopping activeSyncManager {}", e);
```
Comment (gpang) [inline core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java]: for both of these, can we log the mount id and the mount URI?

File: core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java (hunk 3 above)
Comment (yuzhu) [reply core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java]: mountURI involves mount table look up, logged mount id for now.

File: core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java (hunk 4 above)
Comment (yuzhu) [reply core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java]: mountURI involves mount table look up, logged mount id for now.

**Pull request merged**